trio/_tools/gen_exports.py

```diff
@@ -122,6 +159,9 @@ def gen_public_wrappers_source(source_path: Path, lookup_path: str) -> str:
         # Create the function definition including the body
         func = astor.to_source(method, indent_with=" " * 4)
```

Comment:
If it's possible to fix the type annotation formatting that would be great but if not I understand

Reply:
Could make `gen_exports` call `black` after generating even. But definitely a separate PR.